modules/solid_mechanics/include/materials/ReturnMappingModel.h

```diff
@@ -8,61 +8,79 @@
 #define RETURNMAPPINGMODEL_H
 
 #include "ConstitutiveModel.h"
+#include "NewtonIterationSingleVariable.h"
 
 /**
- * One or more constitutive models coupled together.
+ * Base class for models that perform return mapping iterations to compute stress.  This
+ * is for a single model.  This class can be used directly for a single model, or combined
+ * with other models.
  */
 
-class ReturnMappingModel : public ConstitutiveModel
+class ReturnMappingModel : public ConstitutiveModel, public NewtonIterationSingleVariable
```

Comment:
We will follow this same pattern.  RadialReturnStressUpdate will inherit from both StressUpdateBase and NewtonIterationSingleVariable (which I have renamed now), and we'll get rid of all of the Newton iteration code in RadialReturnStressUpdate.  This structure will allow us to accomplish exactly what Andy is suggesting.